sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/RequisitionDao.java

```diff
@@ -11,6 +11,4 @@ public interface RequisitionDao extends PaginatedDataSource<Requisition>, SaveDa
 
   Requisition getByAlias(String alias) throws IOException;
 
-  List<Requisition> listByIdList(Collection<Long> ids) throws IOException;
```

Comment:
all the other files which from extending SaveDao to BulkSaveDao when removing this method, just confirming you don't want to do so here?

Reply:
Yeah, this was intentional. Requisitions don't have a bulk interface and the method was unused